climpred/bootstrap.py

```diff
@@ -498,88 +498,136 @@ def bootstrap_compute(
     comparison = get_comparison_class(comparison, ALL_COMPARISONS)
 
     # Perfect Model requires `same_inits` setup.
-    isHindcast = True if comparison in HINDCAST_COMPARISONS else False
+    isHindcast = True if comparison.name in HINDCAST_COMPARISONS else False
     reference_alignment = alignment if isHindcast else 'same_inits'
 
-    for i in range(iterations):
-        # resample with replacement
-        smp_hind = _resample(hind, resample_dim)
-        # compute init skill
-        init_skill = compute(
-            smp_hind,
-            verif,
-            metric=metric,
-            comparison=comparison,
-            alignment=alignment,
-            add_attrs=False,
-            dim=dim,
-            **metric_kwargs,
+    if hist is None:  # PM path, use verif = control
+        hist = verif
+
+    # slower path for hindcast and resample_dim init
```

Comment:
this is like it was before. needed for hindcast resample_dim init

Reply:
That's right. I think we have to do it this way for `init`.